.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: go
 
 go:
-  - "1.13"
+  - "1.14"
```

Comment:
I have increased go version to go 1.14, I hope that's okay.

Reply:
Absolutely! Thanks :)